### ISSUE TITLE
Characters

### ISSUE THREAD

**Issue by giuli2022** [kviks/Psych-Engine-Android#4]
### What is your question?

Bro, how do I add Characters to the Port? bone Al fnf psych engine

**Comment by Evananggasubagja** [comment]
yes I also don't know how to add characters in the character editor in fnf

**Comment by foxybebe** [comment]
the mod folder is not working
